test/test_data.py

```diff
@@ -263,7 +263,7 @@ def visualtests(self, stream=sys.stdout):
         print("      ", "".join(["%-12s" % pn for pn in parser_names]), file=self.stream)
         print("HOMO", "   ".join(["%+9.4f" % out.moenergies[0][out.homos[0]] for out in output]), file=self.stream)
         print("LUMO", "   ".join(["%+9.4f" % out.moenergies[0][out.homos[0]+1] for out in output]), file=self.stream)
```

Comment:
Missed a few here

Reply:
I was so focused on `.format(...)` that I totally forgot to look for `%` and other stuff. Everything should be covered now.